components/NavBar/index.tsx

```diff
@@ -22,12 +24,14 @@ export const NavBar = () => {
             <HStack spacing="8">
               <Box as="a" href="#" rel="home">
                 <Link passHref href="/">
-                  <span>Hacktobered</span>
+                  <Image 
+                  className={Nav.lo}
+                  src={Hacktober}
+                  width="74vw"
+                  height="65vw"
```

Comment:
Can we also keep width and height in NavBar.module.css?
Also, hope you have checked it on mobile emulators as well.